mod_ci/controllers.py

```diff
@@ -59,6 +60,8 @@ def before_app_request():
         g.user, 'Platform mgmt', 'cog', [], '', [
             {'title': 'Maintenance', 'icon': 'wrench', 'route':
                 'ci.show_maintenance', 'access': [Role.admin]}
```

Comment:
You forgot something, this way the array is not valid ;) Hint, it needs to go on the previous line.